sharepoint-server-farm-ha/nic.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Just curious why you choose to split out these two as subtemplates when they are so small, but not some of the VMs for example.